src/app/notifications/components/ToastNotifications/index.tsx

```diff
@@ -0,0 +1,47 @@
+import flow from 'lodash/fp/flow';
+import orderBy from 'lodash/orderBy';
+import React from 'react';
+import { connect } from 'react-redux';
+import { mobileToolbarOpen } from '../../../content/search/selectors';
+import { AppState } from '../../../types';
+import { assertDefined } from '../../../utils';
+import { dismissNotification } from '../../actions';
+import * as select from '../../selectors';
+import { ToastNotification } from '../../types';
+import { ToastContainerWrapper, ToastsContainer } from './styles';
+import Toast from './Toast';
+
+interface Props {
+  dismiss: (toast: ToastNotification) => void;
+  toasts: ToastNotification[];
+  mobileToolbarOpen: boolean;
+}
+
+// tslint:disable-next-line:variable-name
+const ToastNotifications = (props: Props) => {
+  const realIndexes = new Map(orderBy(props.toasts, ['timestamp'], ['desc']).map((toast, index) => [toast, index]));
```

Comment:
Should this be wrapped in `useMemo` so it's not recalculated on `mobileToolbaOpen` state change?

Reply:
I'm not convinced that it would be faster since you won't get many errors to sort and memo comes with its own cost too